busstops/templates/rules.html

```diff
@@ -30,8 +26,21 @@ <h1>Vehicle editing rules & FAQ</h1>
 <li>
     <p>Don't try to use the <em>edit vehicle details</em> feature to "message" other users or "the admins". If you must, <a href="/contact">contact bustimes.org</a>.</p>
 
+<li>
+    <p>Treat others with respect. Please don't abuse users who approve edits, they are doing the best with the information you supply. If it's declined look at why and find the evidence to back up you're claim.</p>
 </ul>
 
+<h2>Proof</h2>
+
+<p>We ask you to provide proof of your edits in a bid to try and keep the site as correct as possible. Providing proof enables your edits to be approved quicker and reduces issues of you being accued of false edits. Here is our Bustimes handy guide to acceptable proof</p>
+<h3>Acceptable proof</h3>
+<p>Photographs proving what you are saying. You could upload these to another site and provide a link, or if you are struggling, join our <a href="https://discord.com/invite/gY4PdDFau7">Discord</a> community and upload proof there.</p>
+
+<h3>Unacceptable proof</h3>
+<p>Drivers gossip. We love bus drivers but sadly we can't take their word for everything.</p>
+<p>Links to private groups or forums. Sadly our moderators will be unlikely to see your wonderful photographs if they are hidden on closed groups or forums</p>
+<p>Blindly copying information from other "enthusiast websites".</p>
+
 <h2>FAQ</h2>
 
 <h3>Why haven't you designed a perfect system for handling edits, voting, moderation etc?</h3>
```

Comment:
Can you do <details> on these bits so it does a cool drop down thingy like the contact page?